src/k8s/pkg/utils/certificate.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"net"
+)
+
+func SplitIPAndDNSSANs(extraSANs []string) ([]net.IP, []string) {
+	var ipSANs []net.IP
+	var dnsSANs []string
+
+	for _, san := range extraSANs {
+		if san == "" {
```

Comment:
is there a valid use-case for this or does this catch error cases?
If so, we might want to log that an empty SAN was passed.